### ISSUE TITLE
On import, summary ended up with md instead of html in itunes:summary

### ISSUE THREAD

**Issue by kookster** [PRX/cms.prx.org#296]
twww import: https://publish-staging.prx.tech/series/36714
```
        <item>
            <guid isPermaLink="false">f8c5669c265dd96fdedd1a70e615a7f9</guid>
            <title>2.21: 18th and Potomac (with Oliver Platt)</title>
            <pubDate>Wed, 08 Mar 2017 05:41:49 -0000</pubDate>
            <link>https://beta.prx.org/stories/187609</link>
            <description>
                <![CDATA[<p>It's the penultimate episode of Season 2. Oliver Platt (<a href="https://twitter.com/oliverplatt" rel="nofollow" target="_blank">@OliverPlatt</a>) joins Josh and Hrishi to discuss his role as Oliver Babish.</p>

<p> </p>

<p><a href="http://thewestwingweekly.com/episodes/220" rel="nofollow" target="_blank">thewestwingweekly.com/episodes/22</a>1</p>]]>
            </description>
            <enclosure url="http://www.podtrac.com/pts/redirect.mp3/dovetail-staging.prxu.org/westwingweekly/175c5314-6014-4843-a0a1-00588c96d13c/2.21_-_18th_and_Potomac.mp3" type="audio/mpeg" length="85798890"/>
            <itunes:subtitle>It's the penultimate episode of Season 2. Oliver Platt (@OliverPlatt) joins Josh and Hrishi to discuss his role as Oliver Babish. thewestwingweekly.com/episodes/221</itunes:subtitle>
            <itunes:explicit>no</itunes:explicit>
            <itunes:duration>59:30</itunes:duration>
            <itunes:author>Joshua Malina &amp; Hrishikesh Hirway</itunes:author>
            <itunes:summary>
                <![CDATA[It's the penultimate episode of Season 2. Oliver Platt ([@OliverPlatt](https://twitter.com/oliverplatt)) joins Josh and Hrishi to discuss his role as Oliver Babish.

 

 

 

[thewestwingweekly.com/episodes/22](http://thewestwingweekly.com/episodes/220)1]]>
            </itunes:summary>
            <itunes:image href="https://f-staging.prxu.org/westwingweekly/175c5314-6014-4843-a0a1-00588c96d13c/images/5712d8fb-52e7-4b0a-83ce-f2e25618c5f3/TWWW-Logo.png"/>
            <media:content fileSize="85798890" type="audio/mpeg" url="http://www.podtrac.com/pts/redirect.mp3/dovetail-staging.prxu.org/westwingweekly/175c5314-6014-4843-a0a1-00588c96d13c/2.21_-_18th_and_Potomac.mp3"/>
            <content:encoded>
                <![CDATA[<p>It's the penultimate episode of Season 2. Oliver Platt (<a href="https://twitter.com/oliverplatt" rel="nofollow" target="_blank">@OliverPlatt</a>) joins Josh and Hrishi to discuss his role as Oliver Babish.</p>

<p> </p>

<p><a href="http://thewestwingweekly.com/episodes/220" rel="nofollow" target="_blank">thewestwingweekly.com/episodes/22</a>1</p>]]>
            </content:encoded>
        </item>
```

**Comment by kookster** [comment]
yeah, the md is saved into the summary, so weird.
https://feeder-staging.prx.tech/api/v1/episodes/175c5314-6014-4843-a0a1-00588c96d13c

**Comment by kookster** [comment]
found it - in the episode_distribution, setting the summary to the story.description

**Comment by gcampo88** [comment]
closed via #297
